spec/controllers/network_router_controller_spec.rb

```diff
@@ -180,6 +180,24 @@
     end
   end
 
+  describe "#delete_network_routers" do
+    let(:router) { FactoryGirl.create(:network_router, :name => "router-01") }
+    before do
+      controller.instance_variable_set(:@_params, :id => router.id)
+      controller.instance_variable_set(:@lastaction, "show")
+      controller.instance_variable_set(:@layout, "network_router")
+
+      allow(controller).to receive(:find_records_with_rbac).and_return([router])
```

Comment:
@webe745 is this line necessary? Feels wrong to mock the method you're actually replacing.